pkg/apis/nmstate/v1alpha1/v1alpha1_suite_test.go

```diff
@@ -5,9 +5,12 @@ import (
 
 	. "github.com/onsi/ginkgo"
 	. "github.com/onsi/gomega"
+
+	"github.com/onsi/ginkgo/reporters"
 )
 
 func TestUnit(t *testing.T) {
 	RegisterFailHandler(Fail)
-	RunSpecs(t, "API Test Suite")
+	junitReporter := reporters.NewJUnitReporter("junit.api.xml")
```

Comment:
We should make it so it reflects the full path. Don't make me think of names for junit files, and even worse, decoding which was created by which suite. We can just name this one apis-nmstate-v1alpha1-v1alpha1_suite_test.junit.xml

Reply:
Let's just call them junit.xml and copy with full path at artifacts path